examples/first-component/src/components/Room/createConnection.ts

```diff
@@ -0,0 +1,20 @@
+interface createConnectionProps {
+  serverUrl: string;
+  roomId: string;
+}
+
+const createConnection = ({ serverUrl, roomId }: createConnectionProps) => {
+  // A real implementation would actually connect to the server
+  return {
+    connect() {
+      console.log(
+        '✅ Connecting to "' + roomId + '" room at ' + serverUrl + "..."
+      );
+    },
+    disconnect() {
+      console.log('❌ Disconnected from "' + roomId + '" room at ' + serverUrl);
+    },
```

Comment:
ES6 syntax, and create variable for these

Reply:
Fixed